go/vt/wrangler/materializer.go

```diff
@@ -93,6 +93,26 @@ func (wr *Wrangler) addTablesToVSchema(ctx context.Context, sourceKeyspace strin
 	return nil
 }
 
+func shouldInclude(table string, excludes []string) bool {
+	// We filter out internal tables elsewhere when processing SchemaDefinition
+	// structures built from the GetSchema database related API calls. In this
+	// case, however, the table list comes from the user via the -tables flag
+	// so we need to filter out internal table names here in case a user has
+	// explcility specified some.
```

Comment:
typo explcility=>explicitly. Maybe fix later, no need for CI to run again for this!